### ISSUE TITLE
Citing your work

### ISSUE THREAD

**Issue by charlesreid1** [ellisvalentiner/credit-card-fraud#6]
Hi Ellis,

First, thanks for putting your work on GitHub - it has been a challenge to train a neural network on the credit card fraud data set, since most of the networks I've tried insist on being "lazy" and classifying everything as not fraud. It's great to have a working example as a starting point to figure out how to improve what I'm doing.

I was wondering how I can describe the implementation of your network elsewhere (e.g., my [data-engineering-scenarios/kaggle-sql-jupyter-keras](https://github.com/data-engineering-scenarios/kaggle-sql-jupyter-keras) repository, which is illustrating an SQL-to-Keras data pipeline). Did the SReLU architecture/layer idea originally come from you, or was it mentioned in the [Dal Pazollo paper](https://www3.nd.edu/~dial/publications/dalpozzolo2015calibrating.pdf) or some other paper? 

Of course, your work in implementing it in Keras was ultimately the crucial ingredient, and the important step for helping me get some traction on the problem - so thanks again!

**Comment by ellisvalentiner** [comment]
@charlesreid1 sorry I just saw this now

> I was wondering how I can describe the implementation of your network elsewhere

You can cite or describe my work anyway you like. If you would prefer, I'd be happy to make a contribution to your repo

> Did the SReLU architecture/layer idea originally come from you, or was it mentioned in the Dal Pazollo paper or some other paper?

The architecture decisions, e.g. choice of activations (SReLU), were informed by general principles and manual exploration. For example, rectified linear units tend to work well on many problems and S-shaped rectified linear units have two advantages of being able to learn both convex and non-convex functions and can avoid exploding/vanishing gradients.

If I were to implement this now, or if I get around to revising it, I would likely try the new Swish activation function.

> Of course, your work in implementing it in Keras was ultimately the crucial ingredient, and the important step for helping me get some traction on the problem - so thanks again!

Very glad someone found this useful. This itself was a problem I used to teach myself some of these concepts.
